config/diff_test.go

```diff
@@ -355,6 +399,7 @@ func TestDiffConfigs(t *testing.T) {
 
 			for _, revealSensitiveConfigDiffs := range []bool{true, false} {
 				t.Run(fmt.Sprintf("revealSensitiveConfigDiffs=%t", revealSensitiveConfigDiffs), func(t *testing.T) {
+					logger.Infof("Test name: %v LeftFile: `%v` RightFile: `%v`", tc.Name, tc.LeftFile, tc.RightFile)
```

Comment:
I assume you had this for debugging?

Reply:
Yeah, but I left it in intentionally. Even though Go does a good job when using `t.Run` of delineating which test a log line is associated with, I found some additional brief context of the test case was useful. I can remove if you think it's problematic.

Reply:
Feel free to leave in!